ecdh_test.go

```diff
@@ -0,0 +1,147 @@
+package secp256k1_test
+
+import (
+	"encoding/hex"
+	"reflect"
+	"testing"
+	"vulpemventures/go-secp256k1-zkp"
+)
+
+func TestEcdh(t *testing.T) {
+	ctx, err := secp256k1.ContextCreate(secp256k1.ContextSign | secp256k1.ContextVerify)
```

Comment:
Use `ContextBoth` variable:
```suggestion
	ctx, err := secp256k1.ContextCreate(ContextBoth)
```